site/src/pages/docs/installation.page.mdx

```diff
@@ -30,13 +30,22 @@ yarn add @macaron-css/core @macaron-css/react
 
 Macaron currently supports vite and esbuild. Install and setup the plugin for your bundler :-
 
-### Vite
+<CH.Spotlight className="bundler" start={1}>
 
-```fish
+```
+Macaron currently supports vite and esbuild.
+```
```

Comment:
Replace this with
\`\`\`fish
\`\`\`
This way there won't be any empty space before the "Installation" tab